fakestorage/object.go

```diff
@@ -361,6 +372,8 @@ func (s *Server) downloadObject(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Accept-Ranges", "bytes")
 	w.Header().Set("Content-Length", strconv.Itoa(len(content)))
 	w.Header().Set(contentTypeHeader, obj.ContentType)
+	w.Header().Set("X-Goog-Generation", fmt.Sprintf("%d", obj.Generation))
```

Comment:
Let's use strconv instead:

```suggestion
	w.Header().Set("X-Goog-Generation", strconv.FormatInt(obj.Generation, 10))
```